linalg/hypre_parcsr.cpp

```diff
@@ -1560,7 +1560,8 @@ void hypre_ParCSRMatrixAbsMatvecT(hypre_ParCSRMatrix *A,
 
    if (num_cols_offd)
    {
-#if MFEM_HYPRE_VERSION >= 21100
+#if MFEM_HYPRE_VERSION >= 21100 && 0
+      //function below will crash if offdT is on the device
```

Comment:
Can we change this to:

```suggestion
      // Disabled for now, until we implement hypre_CSRMatrixAbsMatvec on device
```